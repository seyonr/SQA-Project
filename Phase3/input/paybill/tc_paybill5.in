login 
standard
99999
Bill Gates
P
EC
2100