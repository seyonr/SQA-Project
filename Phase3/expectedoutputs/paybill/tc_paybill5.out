Welcome to the banking system!
Do you want to login or exit : 
Enter login type (admin or standard): 
Enter account number: 
Enter account name: 
Login successful!

What do you want to do today?
W – Withdrawal
T - Transfer
P - Paybill
D - Deposit
C - Create
DE - Delete
DI - Disable
CH - Change Plan
L – Logout
Please select the payee you want to pay the bill to:
EC - The Bright Light Electric Company
CQ - Credit Card Company Q
FI - Fast Internet, Inc.
How much would you like to pay : 
Standard users can’t pay bills above 2000! 
